Python/other/find_Triplets_with_zero_sum.py

```diff
@@ -0,0 +1,66 @@
+'''
+  Author : Mohit Kumar
+  
+  python program to find triplets in a given  array whose sum is zero 
+
+'''
+
+# function to print triplets with 0 sum 
+
+def findTriplets(arr, n): 
+
+	found = False
+
+	# sort array elements 
+	arr.sort() 
+
+	for i in range(0, n-1): 
+	
+		# initialize left and right 
+		l = i + 1
+		r = n - 1
+		x = arr[i] 
+		while (l < r): 
```

Comment:
Assign x + arr[l] + arr[r] to a variable and then use it in if-else loop.
This will avoid redundant addition operation.